save-cloud-common/src/jvmMain/kotlin/com/saveourtool/save/entitiescosv/LnkVulnerabilityMetadataUser.kt

```diff
@@ -14,9 +14,10 @@ class LnkVulnerabilityMetadataUser(
     // Fetching two or more Bags(List) at the same time on an Entity could form a Cartesian Product. Since a Bag doesn't have an order,
     // Hibernate would not be able to map the right columns to the right entities. Hence, in this case, it throws a MultipleBagFetchException.
     // Link to docs: https://www.baeldung.com/java-hibernate-multiplebagfetchexception#cause-of-multiplebagfetchexception
-
+    @Column(name = "vulnerability_metadata_id")
     var vulnerabilityMetadataId: Long,
 
+    @Column(name = "user_id")
```

Comment:
it does not matched by default?

Reply:
For some reason, after connecting the second database, this stopped working